sdk/python/kfp/dsl/pipeline_context.py

```diff
@@ -189,6 +189,12 @@ def pop_tasks_group(self):
         """Removes the current TasksGroup from the stack."""
         del self.groups[-1]
 
+    def get_last_tasks_group(self) -> Union['tasks_group.TasksGroup', None]:
```

Comment:
Why not just use `Optional[x]` than `Union[x, None]`?

Reply:
Updated.